tsdb/compact.go

```diff
@@ -561,7 +567,7 @@ func (c *LeveledCompactor) write(dest string, meta *BlockMeta, blocks ...BlockRe
 	// data of all blocks.
 	var chunkw ChunkWriter
 
-	chunkw, err = chunks.NewWriter(chunkDir(tmp))
+	chunkw, err = chunks.NewWriterWithSegSize(chunkDir(tmp), c.maxChunkSize)
```

Comment:
```suggestion
	chunkw, err = chunks.NewWriterWithSegSize(chunkDir(tmp), c.maxChunkSegmentSize)
```

and same everywhere would be nice 🤗